Documentation/bpf/toolchain.rst

```diff
@@ -144,7 +144,8 @@ In order to run through all BPF selftests, the following command is needed:
 
     $ sudo make run_tests
 
-If you see any failures, please contact us on Slack with the full test output.
+If you see any failures, please contact us on `Cilium Slack`_ with the full
+test output.
```

Comment:
Who is "us"? I know this is inherited language, but it's an opportunity to be specific and helpful rather than remaining vague.

Reply:
Not part of this PR.